design-system/materials/internals/story/samplers.js

```diff
@@ -0,0 +1,82 @@
+import styled from '@emotion/styled';
+
+const BasicSample = styled.div``;
```

Comment:
This file contains the small components we use to render some examples of different design token values.